ner_v2/detectors/numeral/number/standard_number_detector.py

```diff
@@ -21,18 +24,21 @@ def __init__(self, entity_name, data_directory_path, min_digit, max_digit):
         self.original_date_text = []
         self.entity_name = entity_name
         self.tag = '__' + entity_name + '__'
-        self.min_digit = min_digit
-        self.max_digit = max_digit
 
         self.numbers_word = {}
-        self.language_number_map = {}
+        self.language_scale_map = {}
+        self.units_map = {}
 
         # Method to initialise value in regex
         self.init_regex_and_parser(data_directory_path)
 
+        self.regex_numeric_patterns = re.compile(r'(([\d,]+[.]?[\d]*)\s?(' + "|".join(self.language_scale_map.keys())
+                                                 + r')?)\s*', re.UNICODE)
```

Comment:
Why is the last \s optional?

Reply:
in case of `200-300`, Processed text will be " 200-300 ". It will not match as for 200 as it will be expecting a space instead of always.